shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -84,9 +84,17 @@ showProfileType
     ;
 
 setVariable
-    : SET variable?
+    : SET variableExpr (COMMA_ variableExpr)*
     ;
 
+variableExpr
+    : variable EQ_ variableValue
```

Comment:
Please consider about the relationship of rule name of `variable` and `variableValue`

Reply:
variable is abstract object，include some property